_data/poshan0126/poshan0126.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Poshan Pandey # No longer than 28 characters
+institution: Nepal College of Information Technology 🚩 # no longer than 58 characters
+profile_pic: poshan.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```

Comment:
Incorrect file name, please do one of the following:
1. Change `poshan.jpg` to `git.jpg` in the markdown file.
2. Rename `git.jpg` to `poshan.jpg`.